Zimbra/zm-soap-harness
# ZCS-13336:Added test case to verify user is able to sendAs sendonbeha…

ZCS-13336:Added test case to verify user is able to sendAs and sendonbehalf mails using primary as well as persona
Test cases written:
Test case 1: 	1. Login into Admin.
			2. Grant Permission "sendAs" to Account1 on Account2
			3. Login into Account1.
			4. Send DiscoverRightsRequest to discover grants granted.
			5. Send email with From: Primary Account to Account3 and Verify message details using GetMsgRequest
			6. Verify message details using GetMsgRequest
Test case 2: 
                        1. Login into Admin.
			2. Grant Permission "sendAs" to Account1 on Account2
			3. Login into Account1.
			4. Send DiscoverRightsRequest to discover grants granted.
			5. Create Persona and Send email with From: Primary Account to Account3 and Verify message details using 
			GetMsgRequest
			7. Verify message details using GetMsgRequest
Test case 3: 
			1. Login into Admin.
			2. Grant Permission "sendAs" to Account1 on Account2
			3. Login into Account1.
			4. Send DiscoverRightsRequest to discover grants granted.
			5. Send email with From: Primary Account to Account3 and Verify message details using GetMsgRequest
			6. Verify message details using GetMsgRequest
Test case 4: 
			1. Login into Admin.
			2. Grant Permission "sendAs" to Account1 on Account2
			3. Login into Account1.
			4. Send DiscoverRightsRequest to discover grants granted.
			5. Send email with From: Primary Account to Account3 and Verify message details using GetMsgRequest
			6. Verify message details using GetMsgRequest
